instrumentation/opentelemetry-instrumentation-aiopg/src/opentelemetry/instrumentation/aiopg/aiopg_integration.py

```diff
@@ -86,9 +86,11 @@ def acquire(self):
         async def _acquire(self):
             # pylint: disable=protected-access
             connection = await self.__wrapped__._acquire()
-            return get_traced_connection_proxy(
-                connection, db_api_integration, *args, **kwargs
-            )
+            if not hasattr(connection, "__wrapped__"):
```

Comment:
we probably could check if the wrapper type is the same as the instrumented type but not sure if it's worth it. This is probably enough,.

Reply:
I'd probably have argued for something a bit clearer that it was wrapped by opentelemetry, such as `__wrapped_by_opentelemetry__`. But agreed it's probably overkill: highly unlikely to overlap.

Reply:
I agree with you. `if not isinstance(connection, AsyncProxyObject):` - I think that this option more accurately defines the case of verification.

Reply:
It does but not sure if it is worth the extra check :) Probably we should do what @toumorokoshi suggested in a separate PR so all instrumentations benefit from it,.